CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 UNRELEASED
 ----------
+* [ [#1532](https://github.com/digitalfabrik/integreat-cms/issues/1532) ] Fix cyrillic TOC
```

Comment:
```suggestion

* [ [#1532](https://github.com/digitalfabrik/integreat-cms/issues/1532) ] Fix cyrillic TOC
```
The changelog needs one extra line after the "unreleased" heading and two new lines between the bullet points and the next heading.

Reply:
Oh I forgot this styling again 🙈 Thank you for the correction.